server/routes/shared_api/observation/series.py

```diff
@@ -42,17 +42,23 @@
 bp = Blueprint("series", __name__, url_prefix='/api/observations/series')
 
 
-@bp.route('', strict_slashes=False)
+@bp.route('', strict_slashes=False, methods=['GET', 'POST'])
```

Comment:
use `request.method == 'POST'` to check method and branch logic?

Reply:
done